minorminer/layout/layout.py

```diff
@@ -161,19 +161,18 @@ def _graph_distance_matrix(G,
     if disconnected_distance is None:
         disconnected_distance = len(G)
 
-    return np.array(
-        [
-            [V.get(v, disconnected_distance) for v in sorted(G)]
-                 for u, V in all_pairs_shortest_path_length
-        ]
-    )
+    dist_list = [
+        (u, [V.get(v, disconnected_distance) for v in sorted(G)])
+         for u, V in nx.all_pairs_shortest_path_length(G)
+         ]
+    return(np.array([_[1] for _ in sorted(dist_list)]))
```

Comment:
- missing space
- extra parenthesis
- `_` should be used for throwaway variables only, not here

We try to follow [pep-8](https://peps.python.org/pep-0008/) in Ocean.

Reply:
This should be much simpler once sorting is removed.

Reply:
Done in commit `efb48f1`.